density/density.py

```diff
@@ -17,6 +18,14 @@
 from db import db
 from config import flask_config
 
+
+import pandas as pd
+from data import plot_prediction_point_estimate,df_predict, db_to_pandas_pivot
+import datetime as dt
```

Comment:
Could you rearrange the imports?

The order should be:
```
Python stdlib imports
<blank line>
3rd party imports (e.g. pandas, bokeh, flask)
<blank line>
App-specific imports (e.g. data)
```

with all imports in alphabetical order. So something like:

```
import datetime as dt

from bokeh.embed import components
import pandas as pd
import pscyopg2
```